src/head/get-from-head-data-store.js

```diff
@@ -0,0 +1,28 @@
+(function (M) {
+    const element = document.querySelector('#head-data-store');
+    const data = JSON.parse(element.textContent);
+
+	M.getFromHeadDataStore = function (path) {
+		// TODO split, join, split, this can be optimized
```

Comment:
any plans for resolving this TODO? :)

Reply:
Not really, this method is rarely used so it's no bottleneck. I'll remove it.